CHANGELOG.md

```diff
@@ -1,22 +1,22 @@
-<!--
-SPDX-FileCopyrightText: 2020 Helmholtz Centre for Environmental Research (UFZ)
-SPDX-FileCopyrightText: 2020 Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
+# Changelog
 
-SPDX-License-Identifier: Apache-2.0
--->
+## [Unreleased](https://github.com/hifis-net/ansible-role-haproxy/tree/HEAD)
 
-# Changelog
+[Full Changelog](https://github.com/hifis-net/ansible-role-haproxy/compare/v1.5.0...HEAD)
 
-All notable changes to this project will be documented in this file.
+**Merged pull requests:**
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+- Add badges to README.md [\#3](https://github.com/hifis-net/ansible-role-haproxy/pull/3) ([Normo](https://github.com/Normo))
+- Implement GitHub actions workflows [\#1](https://github.com/hifis-net/ansible-role-haproxy/pull/1) ([Normo](https://github.com/Normo))
 
-Group your changes into these categories:
+<!--
+SPDX-FileCopyrightText: 2020 Helmholtz Centre for Environmental Research (UFZ)
+SPDX-FileCopyrightText: 2020 Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
 
-`Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `Security`.
+SPDX-License-Identifier: Apache-2.0
+-->
 
-## [1.5.0](https://gitlab.com/hifis/ansible/haproxy-role/-/releases/v1.5.0) - 2022-02-22
+## [v1.5.0](https://github.com/hifis-net/ansible-role-haproxy/releases/tag/v1.5.0) - 2022-02-22
 
 [List of commits](https://gitlab.com/hifis/ansible/haproxy-role/-/compare/v1.4.0...v1.5.0)
```

Comment:
Dear Norman @Normo, is it possible to tell the tool to exchange these links as well or do we actually want to link to GitLab here?